x/ibc-transfer/types/packet.go

```diff
@@ -1,10 +1,22 @@
 package types
 
 import (
+	"strings"
+
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 )
 
+const (
+	// DefaultPacketTimeoutHeight is the default packet timeout height relative
+	// to the current block height. The timeout is disabled when set to 0.
+	DefaultPacketTimeoutHeight = 1000 // NOTE: in blocks
+
+	// DefaultPacketTimeoutTimestamp is the default packet timeout timestamp relative
+	// to the current block timestamp. The timeout is disabled when set to 0.
+	DefaultPacketTimeoutTimestamp = 0 // NOTE: in nanoseconds
```

Comment:
we should move these to cli, unless they are used within the types or keeper package?